src/hooks/usePagePath.tsx

```diff
@@ -1,11 +1,12 @@
 import { useLocation } from "@docusaurus/router";
 
-const useDocumentationPath = () => {
+const usePagePath = () => {
```

Comment:
Can it be either documentation or landing? Or is there a third state?

Maybe `usePageType` would be a better name because this hook doesn't return a page path. Or something simillar.